circle.yml

```diff
@@ -11,6 +11,7 @@ test:
     - cd $IMPORTPATH && go test -v -race $(go list ./... | grep -v /vendor/)
     - cd $IMPORTPATH && go vet $(go list ./... | grep -v /vendor/)
     - cd $IMPORTPATH && scripts/end2end_test.sh
+    - cd $IMPORTPATH && go test -run='^$' -bench=. $(go list ./... | grep -v /vendor/)
```

Comment:
if you try and run the benchmarks in circleci and they fail, the circleci job will fail.
eg. https://circleci.com/gh/raintank/metrictank/1166?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link